### ISSUE TITLE
man.vim: first argument disappearing

### ISSUE THREAD

**Issue by nhooyr** [neovim/neovim#5202]
- `nvim --version`: NVIM v0.1.5-725-gb7b870a
- Vim (version: ) behaves differently? no
- Operating system/version: OSX El Capitan
- Terminal name/version: Iterm2 3.0.5
- `$TERM`: xterm-256color

This is a bug with `man.vim` but here is an isolated reproducible case of the same problem.

```vim
nvim -u NORC
:command! -count -nargs=+ Test echo v:count <q-args>
:Test 2 foo
```

The first argument `2` is not printed out for some reason. It’s not in the count or in the arguments. Where did it go? Removing the -count flag fixes it but that flag is needed for `man.vim`.

**Comment by jamessan** [comment]
From `:help :command-count`:

> or as a "count"
argument (-count=N, like the |:Next| command).  The count will then be
available in the argument with |&lt;count&gt;|.

Which means that you would need to do

```vim
:command! -count -nargs=+ Test echo <count> <q-args>
:Test 2 foo
```

**Comment by nhooyr** [comment]
Yea, it's fixed in #5203.
